### ISSUE TITLE
[Inspect]There are no list items displaying under the list tree node in Inspect when expanding the Cursor/Color properties comboBox in PropertyGrid

### ISSUE THREAD

**Issue by ghost** [dotnet/winforms#3027]
**.NET Core Version:**
.NET Core SDK 3.1.300-preview-015048 + VS2019 with latest core designer

**Have you experienced this same bug with .NET Framework?:**
Yes

**Problem description:**
When Inspect focus on the dropdown list value, cannot show the list items.

**More Info:**

1. This issue can reproduce in the .NET Framework project from .Net Framework 4.7 to latest 4.8.
2. This issue can reproduce in the Core 3.1 and Core 5.0 project.
3. This issue can reproduce in the Anchor, BackColor, Dock, ForeColor properties.

**Expected behavior:**
The list node should be expanded and show all list items.
![Expected](https://user-images.githubusercontent.com/53589182/78127836-7012c600-7447-11ea-86b5-51ca94590f5e.png)

**Actual behavior:**
The list node cannot be expand in Inspect, there is no any list items shown.
![Actual](https://user-images.githubusercontent.com/53589182/78127991-ba944280-7447-11ea-97c0-85d22cbf9f7e.png)

**Minimal repro:**
1. Run attached app.
2. Open the inspect tool.
3. Click the Open button next to the Cursor/Color property on the form
4.  Select one item in the dropdown list.
5. Observer the tree structure in Inspect.
[ComboBoxListIssue.zip](https://github.com/dotnet/winforms/files/4414524/ComboBoxListIssue.zip)

**Comment by Zheng-Li01** [comment]
Verified the issue with 5.0 & latest 6.0, the feature has been supported that the list items displaying under the list tree node in Inspect when expanding the Cursor/Color properties comboBox in PropertyGrid, see below screenshot. can we close it?
![image](https://user-images.githubusercontent.com/38325459/125549513-70072217-5fee-4b0c-83dc-3b345817bc37.png)

**Comment by merriemcgaw** [comment]
Yep
